dashel/dashel-win32.cpp

```diff
@@ -1167,7 +1170,7 @@ namespace Dashel
 			if(!readyToRead)
 			{
 				// Block until something happens.
-				WaitForSingleObject(hev, INFINITE);
+				WaitForSingleObject(hev, DEFAULT_WAIT_TIMEOUT);
```

Comment:
why not infinite?

Reply:
hum, yeah, this one could (probably should ) be infinite.